src/components/Button/Button.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon } from '../Icon/Icon';
+
+import './Button.scss';
+
+export const Button = ({
+  toggle,
+  isActive,
+  color,
+  iconName,
+  iconColor,
+  text,
+}) => {
+  const [isDisabledBtn, setIsDisabledBtn] = useState(false);
+
+  useEffect(() => {
+    if (text === 'Pause' && isActive === false) {
+      setIsDisabledBtn(true);
+    } else {
+      setIsDisabledBtn(false);
+    }
+    return () => setIsDisabledBtn(false);
+  }, [isDisabledBtn, isActive, text]);
```

Comment:
To make Button more reusable, we should keep the business logic out. It should only know to disable itself when it is being told (which you have with `isActive`), agnostic of any other prop that is passed in.